.changes/include-image-macro.md

```diff
@@ -0,0 +1,10 @@
+---
+"tauri": "patch:feat"
+"tauri-utils": "patch:feat"
+"tauri-macros": "patch:feat"
+"tauri-codegen": "patch:feat"
+"tauri-cli": "patch:feat"
+"@tauri-apps/cli": "patch:feat"
```

Comment:
```suggestion
"tauri-cli": "patch:feat"
"@tauri-apps/cli": "patch:feat"
```

This change doesn't concern CLI even if `schema.json` changed, same for `tauri-utils`, it is just a documentation change so it doesn't need a bump